guides/front_end/react_quick_start.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: React quick start — Meilisearch documentation
+description: Learn how to integrate a search-as-you-type experience into your React app.
+---
+
+# React quick start
+
+Learn how to integrate a search-as-you-type experience into your React app.
```

Comment:
Not super big, but some technical writers recommend avoiding "learn" because it's a bit presumptuous. I'm not sure how much I agree with that, but thought it was good to mention.